sdk/src/sharedTest/java/io/customer/sdk/CustomerIOClientTest.kt

```diff
@@ -73,13 +81,26 @@ class CustomerIOClientTest : BaseTest() {
         val givenDeviceToken = String.random
         val givenAttributes = mapOf("name" to String.random)
         prefRepository.saveDeviceToken(givenDeviceToken)
-        whenever(backgroundQueueMock.addTask(any(), any())).thenReturn(QueueModifyResult(true, QueueStatus(siteId, 1)))
+        whenever(backgroundQueueMock.addTask(any(), any(), anyOrNull(), anyOrNull())).thenReturn(QueueModifyResult(true, QueueStatus(siteId, 1)))
 
         customerIOClient.identify(newIdentifier, givenAttributes)
 
-        verify(backgroundQueueMock).addTask(QueueTaskType.IdentifyProfile, IdentifyProfileQueueTaskData(newIdentifier, givenAttributes))
-        verify(backgroundQueueMock, never()).addTask(eq(QueueTaskType.DeletePushToken), any())
-        verify(backgroundQueueMock).addTask(QueueTaskType.RegisterDeviceToken, RegisterPushNotificationQueueTaskData(newIdentifier, givenDeviceToken, dateUtilStub.givenDate))
+        inOrder(backgroundQueueMock).apply {
+            verify(backgroundQueueMock).addTask(
+                QueueTaskType.IdentifyProfile,
+                IdentifyProfileQueueTaskData(newIdentifier, givenAttributes),
+                groupStart = QueueTaskGroup.IdentifyProfile(newIdentifier),
+                blockingGroups = null
```

Comment:
`blockingGroups` can be `null` by default?

Reply:
Are you asking why this code is:
```kt
            verify(backgroundQueueMock).addTask(
                QueueTaskType.IdentifyProfile,
                IdentifyProfileQueueTaskData(newIdentifier, givenAttributes),
                groupStart = QueueTaskGroup.IdentifyProfile(newIdentifier),
                blockingGroups = null
            )
```
instead of
```kt
            verify(backgroundQueueMock).addTask(
                QueueTaskType.IdentifyProfile,
                IdentifyProfileQueueTaskData(newIdentifier, givenAttributes),
                groupStart = QueueTaskGroup.IdentifyProfile(newIdentifier)
            )
```
(removing `blockingGroups = null`) because `blockingGroups` is null by default?

Reply:
oh never mind, I thought it's not `null` by default, but just checked the code, it indeed is `null by default`. You just added it out of practice.